packages/esm-shr-app/src/shrpatient-summary/shrAllergiesDataTable.component.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useMemo } from 'react';
+import styles from './shr-summary.scss';
+import { useTranslation } from 'react-i18next';
+import { DataTable, Table, TableHead, TableRow, TableHeader, TableBody, TableCell } from '@carbon/react';
+import { itemDetails } from '../types';
+import { usePagination } from '@openmrs/esm-framework';
+import { PatientChartPagination } from '@openmrs/esm-patient-common-lib';
+
+interface SHRDataTableProps {
+  data: itemDetails[];
+}
+
+const SHRAllergiesDataTable: React.FC<SHRDataTableProps> = ({ data }) => {
+  const { t } = useTranslation();
+  const tableHeaders = ['Allergen', 'Reaction', 'Severity', 'Onset Date', 'Date Recorded'];
```

Comment:
```suggestion
  const tableHeaders = [{key:'allergen', t('allergen','Allergen')}];
```
the `t` function is to provide translation and carbon definition of headers should have the key and header values [see](https://react.carbondesignsystem.com/?path=/docs/components-datatable-basic--overview)

Reply:
Done